src/adaptors/tcp/tcp_adaptor.c

```diff
@@ -2247,24 +2236,23 @@ static void qdr_tcp_adaptor_init(qdr_core_t *core, void **adaptor_context)
 {
     qdr_tcp_adaptor_t *adaptor = NEW(qdr_tcp_adaptor_t);
     adaptor->core    = core;
-    adaptor->adaptor = qdr_protocol_adaptor(core,
-                                            "tcp",                // name
-                                            adaptor,              // context
-                                            qdr_tcp_activate_CT,  // runs on Core thread
-                                            qdr_tcp_first_attach,
-                                            qdr_tcp_second_attach,
-                                            qdr_tcp_detach,
-                                            qdr_tcp_flow,
-                                            qdr_tcp_offer,
-                                            qdr_tcp_drained,
-                                            qdr_tcp_drain,
-                                            qdr_tcp_push,
-                                            qdr_tcp_deliver,
-                                            qdr_tcp_get_credit,
-                                            qdr_tcp_delivery_update,
-                                            qdr_tcp_conn_close,
-                                            qdr_tcp_conn_trace);
-    adaptor->log_source  = qd_log_source("TCP_ADAPTOR");
+    adaptor->adaptor           = qdr_protocol_adaptor(core,
```

Comment:
What's up with this?

Reply:
That is because of the clang-format. I will go thru this PR and make sure everything is ok. Sorry about that.
"I will fix what clang broke" is something I never thought I would say.

Reply:
I think I have now fixed all of the formatting issues introduced by clang-format